mne_bids/copyfiles.py

```diff
@@ -20,10 +20,12 @@
 
 import shutil as sh
 from datetime import datetime
+from scipy.io import savemat
```

Comment:
Personally I like the Python convention to have 3 import blocks: 1) builtins, 2) external, 3) local --> don't know which pep that is, but pretty sure it's part of some official style. So I would keepscipy listed near MNE (both "external")